src/actions/assets/OGC3dTiles.js

```diff
@@ -0,0 +1,55 @@
+import AssetActionPlugin from "../AssetActionPlugin";
+import URI from 'urijs';
+import i18n from "../../i18n";
+
+// See mime types discussion for 3d-tiles here and there
+// https://github.com/opengeospatial/ogcapi-3d-geovolumes/issues/13
+const OGC3DTILES_SUPPORTED_TYPES = [
+  // 'application/json',
+  'application/3dtiles+json',
+];
+
+export default class CesiumSandcastle extends AssetActionPlugin {
+
+  get show() {
+    return this.component.isBrowserProtocol && OGC3DTILES_SUPPORTED_TYPES.includes(this.asset.type);
+  }
+
+  get uri_CesiumSandcastle() {
+    let uri = new URI("https://sandcastle.cesium.com/index.html");
+    const tileset_url = this.component.href;
+    const code_payload = {
+      html: `
+        <style> @import url(../templates/bucket.css); </style>
+        <div id="cesiumContainer" class="fullSize"></div> 
+      `, 
+      code: `
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          terrain: Cesium.Terrain.fromWorldTerrain(),
+        });
+
+        try {
+          const tileset = await Cesium.Cesium3DTileset.fromUrl('${tileset_url}');
+          viewer.scene.primitives.add(tileset);
+          viewer.zoomTo(tileset);
+        } catch (error) {
+          console.log('Error loading tileset');
+        } 
+      `.replaceAll('        ', '')
+    };
+    const code_str = btoa(JSON.stringify(code_payload));
+    uri.addQuery('code', code_str);
+    return uri;
+  }
```

Comment:
@jo-chemla What is this section doing? It doesn't look like it's used anywhere.

Also, why is this called Cesium Sandcastle but it's using Geofox.ai?

Reply:
Sorry I just remembered I switched from Cesium Sandcastle - which is more of a way to test cesium code, although mature and active for years - to the geofox.ai viewer - which looks more like a standalone viewer, focused on loading OGC 3D-tiles tilesets.

Reply:
Okay, so the code should be removed in both links and assets (for now)?!

Reply:
Yes indeed, will do!